source/linux/Foundational_Components/Power_Management/pm_wakeup_sources.rst

```diff
@@ -399,48 +403,138 @@ MCU GPIO
    :ref:`LPM section<lpm_modes>`, wakeup from MCU_SPI0_D1 can be triggered
    by grounding Pin 4 on J8 MCU Header.
 
+*********
+WKUP GPIO
+*********
+
+.. ifconfig:: CONFIG_part_variant in ('AM62LX')
+
+   One of the most common ways to wakeup a system is by using some I/O activity.
+   I/O activity on the WKUP GPIOs can wakeup the system when the WKUP GPIO
+   controller is configured as a wakeup source. Refer to the wkup_gpio_key node
+   in
+   `k3-am62l3-evm-lpm-wkup-sources.dtso <https://git.ti.com/cgit/ti-linux-kernel/ti-linux-kernel/tree/arch/arm64/boot/dts/ti/k3-am62l3-evm-lpm-wkup-sources.dtso?h=11.02.06>`__
+   to use as a template to configure the desired WKUP GPIO as a wakeup capable
+   GPIO.
+
+   A brief guide to configuring an WKUP GPIO as wakeup:
```

Comment:
This should probably be a numbered list, considering the way it's broken up and the lack of complete sentences.